openslides/core/management/commands/collectstatic.py

```diff
@@ -27,7 +27,7 @@ def handle(self, **options: Any) -> str:
 
     def collect(self) -> Dict[str, Any]:
         try:
-            destination_dir = os.path.join(settings.STATICFILES_DIRS[0], 'js')
+            destination_dir = os.path.join(settings.STATIC_ROOT, 'js')
```

Comment:
@FinnStutzenstein Can you check if this is the best solution for OpenSlides core?

Reply:
@normanjaeckel May have a look

Reply:
I do not understand this solution. The command should collect the statics into the collected static dir and not into anything else.

Reply:
@normanjaeckel In my estimation, STATICFILES_DIRS[0] is the first directory where static files should be collected from. STATIC_ROOT is the directory to collect files into! See https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-STATIC_ROOT and https://docs.djangoproject.com/en/2.0/ref/settings/#staticfiles-dirs ... do you come to the same conclusion?

Reply:
Hm. You are right. I mixed it up.

But the problem is, now the extra flags of the collectstatic command won't work properly (like --dry-run, --clear, --link, --ignore) and the STATIC_ROOT dir will get some stuff even if the user doesn't want this.

Reply:
The problem is probably that the whole server/client code mix-up is a hack :-D
Do you need the features dry-run, clear etc?

Reply:
@normanjaeckel Any concerns here? We need a solution that we can merge it soon for using with new multi-backend.

Reply:
I wrote a last question to @boehlke via email to fully understand the problem.